crates/bevy_asset/macros/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+// FIXME(3492): remove once docs is ready
```

Comment:
```suggestion
// FIXME(3492): remove once docs are ready
```

Nit: "docs" is plural in English, unlike "documentation", which is singular. Same elsewhere.